generator/transit_generator.hpp

```diff
@@ -83,6 +83,10 @@ class DeserializerFromJson
 /// \param mwmPath relative or full path to an mwm. The name of mwm without extension is considered
 /// as country id.
 /// \param transitDir a path to directory with json files with transit graphs.
+/// \note An mwm pointed by |mwmPath| should contain:
+/// * section which is necessary to register it at Index
+/// * index graph (ROUTING_FILE_TAG) and restriction sections (RESTRICTIONS_FILE_TAG)
+/// This information will be used in this method.
```

Comment:
The last comment provides zero bits of information.

Reply:
Removed.